nautobot/ipam/forms.py

```diff
@@ -493,6 +493,8 @@ def clean_namespace(self):
         setattr(self.instance, "_namespace", namespace)
 
     def clean(self):
+        # Pass address to the instance, because this is required to be accessible in the IPAddress.clean method
+        self.instance.address = self.cleaned_data.get("address")
```

Comment:
Is this specifically for "create", for "edit", or both cases that this is needed?

Reply:
Both cases, cause address might/can be updated